File: Test_CPA6.java
import java.util.Scanner;

public class Test_CPA6{
    public static void main(String[]args){
      
        System.out.print("Please enter a number: "); //asks for input from the user
        Scanner ix = new Scanner(System.in);
        String  n = ix.next();
       
        if(n.length() > 0){ // checks the length of the input
            CPA6_functions Run = new CPA6_functions(n);
            Run.find_sum(n);
            ix.close();
        } 
        else{ // empty string denotes void input
            System.out.println("your input is void, please enter a number");//if void input, this statement is displayed
        }
    }
}

File: test_CPA6.py
from CPA6_functions import find_sum
def main():
    n = input("please enter a number: ")
    if len(n) > 0: # the function is only invoked if input is not void
        print(find_sum(n))
        
    else: # empty string denotes void input
        print('please enter a number, your input is void')
main()